tests/ut/test_ivfflat_cc.cc

```diff
@@ -74,8 +78,12 @@ TEST_CASE("Test Build Search Concurrency", "[Concurrency]") {
                     std::mt19937_64 rng(i);
                     int64_t add_size = distribution(rng);
                     std::vector<faiss::Index::idx_t> ids(add_size, i);
-                    std::vector<uint8_t> codes(add_size * code_size, (uint8_t)(i % 256));
-                    invList->add_entries(i, add_size, ids.data(), codes.data());
+                    float value = i;
+                    std::vector<float> codes(add_size * dim, value);
```

Comment:
why the data in "codes" are all the same ?

Reply:
This is just to check codes conveniently. Random codes is not easy to check for the ivf hash function.